keyboards/kinesis/rules.mk

```
@@ -1,5 +1,5 @@
 # MCU name
-MCU = at90usb1286
+MCU = atmega32u4
```

This file should remain unchanged, and you should override the necessary rules in `keyboards/kinesis/nguyenvietyen/rules.mk` instead.

```suggestion
MCU = at90usb1286
```

---

Done 👍